src/build/arg/mod.rs

```diff
@@ -4888,7 +4860,7 @@ impl<'help> Display for Arg<'help> {
                 f,
                 "<{}>{}",
                 self.name,
-                if self.is_set(ArgSettings::MultipleOccurrences) {
+                if self.is_set(ArgSettings::MultipleValues) {
```

Comment:
`self.is_set(ArgSettings::MultipleValues) || self.is_set(ArgSettings::MultipleOccurrences)`?

Reply:
With my next commit, positionals now forbid `multiple_occurrences` and take only `multiple_values`.